config/elasticsearch/mappings/reactions.json

```diff
@@ -78,9 +78,6 @@
             },
             "profile_image_90": {
               "type": "keyword"
-            },
-            "pro": {
-              "type": "boolean"
```

Comment:
Our ES docs made it sound like it is safe/acceptable to edit these mapping files directly, so I went ahead and removed the `pro` key from them. Let me know if I misunderstood and actually shouldn't be touching these files for some reason!